sway/tree/layout.c

```diff
@@ -100,14 +101,16 @@ void container_add_child(struct sway_container *parent,
 			parent, parent->type, parent->width, parent->height);
 	list_add(parent->children, child);
 	child->parent = parent;
+	// TODO: set focus for this container?
+	sway_input_manager_set_focus(input_manager, child);
 }
 
 struct sway_container *container_reap_empty(struct sway_container *container) {
-	if (!sway_assert(container, "reaping null container")) {
```

Comment:
Is this now okay? We should assert if it's a programming error to pass in a NULL container